vms/event/services.py

```diff
@@ -89,3 +89,15 @@ def get_events_by_date(start_date, end_date):
 def get_events_ordered_by_name():
     event_list = Event.objects.all().order_by('name')
     return event_list
+
+
+def remove_empty_events(event_list):
+    """ Removes all events from an event list without jobs or shifts """
+    
```

Comment:
There should be no blank line either before or after the docstring. Can you please remove this @ahmedsabie?